### ISSUE TITLE
Header - Add clickable/active areas

### ISSUE THREAD

**Issue by BogdanDogaru** [bbc/simorgh#346]
* This issue applies CSS for the SVG logo and its surrounding anchor element.
* The logo must be visually aligned to the centre
* The logo receives the width of 154px and height of 22px
* The logo needs to be horizontally aligned by 8px
* The touch target area vertically encompasses the height of the header.
* provide an :active state.  border-bottom: 4px solid #FFFFFF;  already implemented for (:hover and :focus states)

**Acceptance Criteria**
> Given I view the Header in GEL Group B (<=599px)
> When I tap on the brand logo
> And it has a width or height of less than 32px
> Then it must have an exclusion zone of at least 32px x 32px

> Given I view the Header in GEL Group D (>=600px)
> When I tap on the brand logo
> And it has a width or height of less than 24px
> Then it must have an exclusion zone of at least 24px x 24px

**Describe the solution you'd like**
We need to style the logo in the Header component to match the UX&D designs referenced in https://jira.dev.bbc.co.uk/browse/NEWSART-594

**Additional context**
GEL tap targets and exclusion zones: https://www.bbc.co.uk/gel/guidelines/how-to-design-for-touch#tap-targets

<img width="810" alt="Screenshot" src="https://user-images.githubusercontent.com/3028997/43261770-cc072020-90d5-11e8-8bdd-685c4bae35ca.png">

The following accessibility testing instructions apply for this issue:
* operable: The anchor should be able to reached / activated by keyboard only
* VoiceOver iOS: Give the SVG VoiceOver focus. Visual styling must remain consistent

**Comment by radiocontrolled** [comment]
Testing for this ticket should include use with VoiceOver on mobile phones to validate that when the SVG receives focus from VoiceOver, that visual styling remains consistent.

**Comment by PaulChinkwende** [comment]
We also need to vertically centre the svg. Could be done by adding a margin to the top and bottom of 4px.

**Comment by radiocontrolled** [comment]
Some questions to be confirmed: 

* does our present markup accommodate how the width of the touch target area (https://jira.dev.bbc.co.uk/secure/attachment/646099/Screen%20Shot%202018-07-19%20at%2011.16.36.png) is larger than that of the anchor & SVG itself (and its underline) (https://jira.dev.bbc.co.uk/secure/attachment/646100/Screen%20Shot%202018-07-19%20at%2011.16.42.png)

* width of the touch target area: is it the same at all viewport sizes? 

* does this issue provide an `:active` state for the anchor? If yes, does active have the `border-bottom: 4px solid #FFFFFF;` of the `:hover` and `:focus` states

**Comment by jtart** [comment]
Also resolves #354.

**Comment by sareh** [comment]
Talking with @nataliesmart about this: the logo should have a height of 24px. The width should be whatever value keeps it having the correct ratio for the logo.  Note that pinning the width to a particular pixel value mentioned in designs isn't necessarily, since when we get to World Service sites e.g. Persian, etc., the width of logos will be variable. The touch area should be sufficiently wide to meet the GEL touch target requirements. https://www.bbc.co.uk/gel/guidelines/how-to-design-for-touch#tap-targets
